src/drawer/style.ts

```diff
@@ -15,20 +15,20 @@ export default css`
     overflow-y: hidden;
   }
 
-  .kirk-drawer--open::after {
+  .kirk-drawer::after {
     content: '';
+    font-size: 0;
+  }
+
+  .kirk-drawer--open::after {
     position: absolute;
     top: 0;
     left: 0;
     bottom: 0;
     right: 0;
     z-index: 1;
-    background-color: rgba(0, 0, 0, 0);
-    transition: background-color 150ms linear;
-  }
-
-  .kirk-drawer--open::after {
     background-color: rgba(0, 0, 0, 0.3);
+    transition: background-color 300ms linear;
```

Comment:
If I may intervene here, should this `300ms` duration go in the `branding` file?

Reply:
We already have `transition: background-color ${transition.duration.fast} linear;` for example but it setted to `200ms`.

Reply:
Yes can you use one of the value in the branding file? Or if it does'n fit your case add a new one
https://github.com/blablacar/ui-library/blob/master/src/_utils/branding.ts#L82

Reply:
Updated to ${transition.duration.base} , thanks!